src/worker/core/season/doAwards.football.ts

```diff
@@ -275,7 +275,29 @@ const doAwards = async (conditions: Conditions) => {
 	});
 	const players = await getPlayers(g.get("season"));
 	const { bestRecord, bestRecordConfs } = await teamAwards(teams);
-	leagueLeaders(players, [], awardsByPlayer);
+	const categories = [
+		{
+			name: "League Passing Leader",
+			stat: "pssYds",
+			minValue: 2000,
```

Comment:
No point in a minimum for pure counting stats... it's only needed for per game stats, but none of these are per game.

Reply:
This is also why it doesn't work, because the minValue checking code assumes the stats in the player objects are per game, but that's only true in BBGM.

But set it to 0 and it still fails. Why? Because the minValue checking code also assumes "gp" is present, which is not true for hockey and football.

Recommendation - get rid of minValue here, and then just skip the minValue checking code in the `leagueLeaders` function (the filter inside the loop) if minValue is undefined.

Or let me know if you want me to just take it from here, I'm fine either way :)

Reply:
BTW this is all happening because this part of the codebase is not properly typed (`players` is `any[]`)